bitrise.yml

```diff
@@ -1062,9 +1063,9 @@ workflows:
                 set -x
 
                 mkdir -p /bitrise/src/trace-test-application/build/reports
-                adb logcat -d > $BITRISE_SOURCE_DIR/trace-test-application/build/reports/logcat.txt &
+                adb logcat -d > $BITRISE_SOURCE_DIR/trace-test-application/build/reports/logcat.txt
       - script@1:
-          title: save_screenshots
+          title: save_collected_test_data
```

Comment:
Updated step to save the captured view hierarchy items.

Reply:
might be worth adding a description element to explain what this test data is, and where it's saving it to?

Reply:
Sure, added it.